src/mrinufft/operators/base.py

```diff
@@ -365,6 +365,32 @@ def get_lipschitz_cst(self, max_iter=10, **kwargs):
             tmp_op = self
         return power_method(max_iter, tmp_op)
 
+    def cg(self, kspace_data, compute_loss=False, **kwargs):
+        """Conjugate Gradient method to solve the inverse problem.
+
+        Parameters
+        ----------
+        kspace_data: np.ndarray
+            The k-space data to reconstruct.
+        computer_loss: bool
+            Whether to compute the loss at each iteration.
+            If True, loss is calculated and returned, otherwise, it's skipped.
+        **kwargs:
+            Extra arguments to pass to the conjugate gradient method.
+
+        Returns
+        -------
+        np.ndarray
+            Reconstructed image
+        np.ndarray, optional
+            array of loss at each iteration, if compute_loss is True.
+        """
+        from ..extras.gradient import cg
+
+        return cg(
+            operator=self, kspace_data=kspace_data, compute_loss=compute_loss, **kwargs
+        )
```

Comment:
Hmm, this is new, i am okay with this also. @paquiteau thoughts?

Reply:
I am fine with this, having the `base.py` file remaining abstract and the implementation in the extra module is neat and tidy for me :)